docs/design/database.md

```diff
@@ -0,0 +1,133 @@
+Database Schema
+===============
```

Comment:
I think markdown lint will complain about this. It doesn't like mixing heading styles (underlining vs. the `#` prefixes)

Reply:
Ack. Fixing now.